src/__tests__/recorder.test.ts

```diff
@@ -29,6 +30,22 @@ describe(recorder.record, () => {
     const [[call]] = jest.mocked(Recording.prototype.functionCall).mock.calls;
     expect(call).not.toHaveProperty("this_");
   });
+
+  it("pauses and resumes recorder", () => {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    const f = () => {};
+    const fInfo = createTestFn("testFun", "param0", "param1");
+
+    pauseRecorder();
+
+    recorder.record.call("this", f, ["arg1", "arg2"], fInfo);
+    expect(Recording.prototype.functionCall).toBeCalledTimes(0);
+
+    resumeRecorder();
+
+    recorder.record.call("this", f, ["arg1", "arg2"], fInfo);
+    expect(Recording.prototype.functionCall).toBeCalledTimes(1);
+  });
```

Comment:
This is great, a unit test is nice to have, but I meant the test for the problem that required this fix :) sorry if I wasn't 100% clear.

You can for example add a `get message()` to an error class in one of the integration tests (add a comment to say why it's there, so nobody will be tempted to remove it); this would have caused the problem before this change, but now it will work.